libcaliptra/inc/caliptra_types.h

```diff
@@ -52,3 +52,14 @@ struct caliptra_fips_version {
     uint8_t name[12];
 };
 
+struct caliptra_stash_measurement_req {
+    caliptra_checksum checksum;
+    uint8_t           metadata[4];
+    uint8_t           measurement[48];
+    uint32_t          svn;
+};
+
+struct caliptra_stash_measurement_cpl {
```

Comment:
nit: I would recommend aligned with https://github.com/chipsalliance/caliptra-sw/blob/main/common/src/mailbox_api.rs and using `resp` instead of `cpl` as a prefix

Reply:
Yeah I was thinking about that. Just pushed up a change that aligns them.

I'll do this on my pending branch too.